czkawka_core/src/duplicate.rs

```diff
@@ -1600,3 +1612,61 @@ fn load_hashes_from_file(text_messages: &mut Messages, type_of_hash: &HashType)
     text_messages.messages.push("Cannot find or open system config dir to save cache file".to_string());
     None
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::fs::{read_dir, File};
+    use std::io::Result;
+    #[cfg(target_family = "windows")]
+    use std::os::fs::MetadataExt;
```

Comment:
It should be `use std::os::windows::fs::MetadataExt;`, but the import (only on Windows) is unused - the `assert_inode` function is empty on Windows and it's the only place, where `MetadataExt` is used (inode number check).

Reply:
I suppose you could do a pull request.